docs/public-networks/concepts/node-sync.md

```diff
@@ -108,6 +117,11 @@ You need Besu version 22.4.3 or later to use checkpoint sync.
 Checkpoint sync behaves like [snap sync](#snap-synchronization), but instead of syncing from the
 genesis block, it syncs from a specific checkpoint block configured in the [Besu genesis file](genesis-file.md).
 
+:::info important
+When you run a fresh checkpoint sync, Besu skips all pre-merge PoW blocks and discards
+their headers. If you need the headers, use Snap sync. If you need the block bodies, use Snap sync with [`--snapsync-synchronizer-pre-checkpoint-headers-only-enabled`](../reference/cli/options.md#snapsync-synchronizer-pre-checkpoint-headers-only-enabled) set to `false`
```

Comment:
I think we could more strongly recommend people use SNAP over CHECKPOINT as we would like to deprecate CHECKPOINT (though haven't yet)